packages/react/src/components/Footer/FooterNav.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
+import { Accordion } from 'carbon-components-react';
+import classNames from 'classnames';
+
+import FooterNavGroup from './FooterNavGroup';
+
+const { prefix } = settings;
+
+/** Footer nav component */
+class FooterNav extends React.Component {
+  /**
+   * Renders the footer nav component
+   *
+   * @returns {object} JSX object
+   */
+  render() {
+    const { groups } = this.props;
+
+    return (
+      <nav className={`${prefix}--footer-nav`}>
```

Comment:
@photodow can you add in stable selectors here? 
https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/docs/stable-selectors.md